spec/features/shelters/users_can_see_shelter_details_spec.rb

```diff
@@ -20,4 +20,22 @@
         expect(page).to have_content(shelter_1.state)
         expect(page).to have_content(shelter_1.zip)
     end
+    
+    it "has a link to their pets" do
+        shelter_1 = Shelter.create( name:    "4 Paws Rescue",
+                            address: "6567 W Long Dr.",
+                            city:    "Littleton",
+                            state:   "CO",
+                            zip:     "80123")
+        
+        visit "/shelters/#{shelter_1.id}"
+        click_link "View Pets"      
+        expect(current_path).to eq("/shelters/#{shelter_1.id}/pets")
```

Comment:
Tests that the shelter has a view pets link that takes the user to the shelter pets page.